packages/editor/src/components/properties/PortalNodeEditor.tsx

```diff
@@ -119,7 +119,10 @@ export const PortalNodeEditor: EditorComponentType = (props) => {
   return (
```

Comment:
while we're here, we should also make `getComponent(entity, PortalComponent) instead useComponent`